### PR TITLE
Update 20-raspibolt-welcome System Overview

### PR DESCRIPTION
**Pull request by lucasmoten**
- Updated to 3 column layout using top area for resources
- Designed to support 80 character width displays
- Parse up load avg, uptime, etc for cleaner look
- Shows bitcoin blockheight
- Includes running state for Electrum and Block Explorer
- Only calls git for bitcoin and lnd release versions once per day

### CODE REVIEW

**Comment by lucasmoten** [comment]
Hold on this as I found some bugs regarding disk size and some error outputs

**Comment by lucasmoten** [comment]
Identified bugs resolved and PR ready

**Comment by Stadicus** [comment]
Nice, thanks a lot. Will try this out soon. Could you add a screenshot here for reference?

**Comment by lucasmoten** [comment]
One of the files in this PR is the updated screenshot

**Comment by lucasmoten** [comment]
Incidentally, I do have a mod to this that 
- resolves a formatting issue on sat sum
- adds version check for electrs

I can update the PR with that shortly.

File: resources/20-raspibolt-welcome [hunk 1]
```diff
@@ -137,16 +156,56 @@ else
...
+else
+  gitupdate=`find ${gitstatusfile} -mtime +1|wc -l`
+fi
+if [ "${gitupdate}" -eq "1" ]; then
+  # Calls to github
+  btcgit=$(curl -s https://api.github.com/repos/bitcoin/bitcoin/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+  lndgit=$(curl -s https://api.github.com/repos/lightningnetwork/lnd/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+  # Electrs and RPC Explorer dont have a latest release, just tags
+  electrsgit=$(curl -s https://api.github.com/repos/romanz/electrs/tags | jq -r '.[0].name')
+  btcrcpexplorergit=$(cur -s https://api.github.com/repos/janoside/btc-rpc-explorer/tags | jq -r '.[0].name')
```
Comment (lucasmoten) [inline resources/20-raspibolt-welcome:173]: doh. I spelled curl wrong here

**Pull request merged**